stdlib/2/UserString.pyi

```diff
@@ -62,6 +62,10 @@ class UserString(Sequence[UserString]):
     def zfill(self: _UST, width: int) -> _UST: ...
 
 class MutableString(UserString, MutableSequence[MutableString]):
+    @overload
+    def __getitem__(self, i: int) -> MutableString: ...
+    @overload
+    def __getitem__(self, s: slice) -> MutableSequence[MutableString]: ...
```

Comment:
Wouldn't this still return a MutableString? And shouldn't this use a self-type, since presumably subtypes of MutableString would return their own type?

Reply:
Changing that to return `_MST` results in the following
```
stdlib/2/UserString.pyi:64: error: Definition of "__getitem__" in base class "UserString" is incompatible with definition in base class "MutableSequence"
stdlib/2/UserString.pyi:65: error: Signature of "__getitem__" incompatible with supertype "UserString"
stdlib/2/UserString.pyi:65: error: Signature of "__getitem__" incompatible with supertype "MutableSequence"
stdlib/2/UserString.pyi:65: error: Signature of "__getitem__" incompatible with supertype "Sequence"
```